# HG changeset patch
# User Rik <rik@octave.org>
# Date 1523560294 25200
#      Thu Apr 12 12:11:34 2018 -0700
# Node ID b68abed2db2082e6505c967677804ef99ec1353d
# Parent  920adb4051a3ba26e1c24d46860ecc85c1e25582
Update RELEASE.PROCESS checklist.

* RELEASE.PROCESS: Update checklist.

diff --git a/etc/RELEASE.PROCESS b/etc/RELEASE.PROCESS
--- a/etc/RELEASE.PROCESS
+++ b/etc/RELEASE.PROCESS
@@ -1,16 +1,14 @@
 <!----------------------------------------------------------------------------->
 <! This file contains a list of steps to accomplish before producing a release.
    The format of the file is wiki markup and can be directly used as a page
    on wiki.octave.org.
 <!----------------------------------------------------------------------------->
-==4.4.0 Release Tasks==
-
-This page shows the tasks to be completed before the GUI release is finalized.
+==5.0.0 Release Tasks==
 
 <!----------------------------------------------------------------------------->
 # Update gnulib to latest version
 # Must occur first as it could resolve existing, or create new, bug reports
 #: Completion Date:
 <!----------------------------------------------------------------------------->
 # File bug reports for all outstanding bugs known, but not reported
 #* Put out a general call for reports on Octave-Maintainers and Octave-Help list
@@ -20,33 +18,33 @@ This page shows the tasks to be complete
 #: Completion Date:
 <!----------------------------------------------------------------------------->
 # Identify Bugs which *must* be fixed prior to release
 #* Review bugs on tracker for possible inclusion in list
 #* Review bugs and update to correct category, such as Patch submitted
 #: Completion Date:
 <!----------------------------------------------------------------------------->
 # Clear all bugs identified as must-fix
-#* See [[Bug Fix List - 4.2 Release]]
+#* See [[Bug Fix List - 5.0 Release]]
 #: Completion Date:
 <!----------------------------------------------------------------------------->
 # GPL License activities
 #* Update Copyright statements for all source controlled files
-#* Update dates in any other locations (launch message, citation, etc.)
+#* Update dates in any other locations (launch message, citation, MXE files, etc.)
 #* Add any new contributors to contributors.in
 #: Completion Date:
 <!----------------------------------------------------------------------------->
 # Style-check code base
 #* This will produce lots of whitespace changes, but no behavior changes
-#* Must occur after patches have been added since whitespace changes will often prevent patches from applying
+#* Must occur after patches have been added since whitespace changes can prevent patches from applying
 #* m-file style check. Completion Date:
 #* C++ style check.  Completion Date:
 <!----------------------------------------------------------------------------->
 # Run lint checker on code base
-#* cppcheck, -fsanitize, etc.
+#* cppcheck, -fsanitize=undefined, etc.
 #: Completion Date:
 <!----------------------------------------------------------------------------->
 # Verify 'make check' is passing on all buildbot combinations of OS and compilers
 #* Start discussion on octave-maintainers list about which failing tests must be fixed
 #* Identify and fix any tests determined critical in step above
 #: Completion Date:
 <!----------------------------------------------------------------------------->
 # Compile and run Octave test suite with --enable-address-sanitizer-flags to check for memory leaks
@@ -70,26 +68,29 @@ This page shows the tasks to be complete
 # Verify build process and create release candidates
 #* Update version information in configure.ac/Makefile.am
 #* Verify 'make distcheck' passes
 #* Create release candidate
 #** 'make dist'
 #** hg tag repository with release candidate ID
 #** For Windows, create installer [[Windows Installer]]
 #** Upload release candidate
+#** add release candidate version to Savannah bug tracker
 #** Announce release candidate to Octave-Maintainers, Octave-Help, on web page
 #** Repeat release candidate cycle until clean
 #: Completion Date:
 <!----------------------------------------------------------------------------->
 # Final Release
 #* hg tag repository with release
 #* merge default onto stable to become the current stable release
 #* add new release version to Savannah bug tracker
 #* Announce final release on Octave mailing lists and web site
 #: Completion Date:
 <!----------------------------------------------------------------------------->
 # Post-Release
 #* Update configure.ac/Makefile.am versioning to next release cycle
 #* Remove all deprecated functions scheduled for deletion in default branch
+#* Move NEWS file to backup in etc/NEWS.X
+#* Create new NEWS file
 #: Completion Date:
 
 [[Category:Releases]]
 [[Category:Development]]
